output/analysis/plots/sconscript.log

```diff
@@ -1,8 +1,74 @@
-*** Builder log created: {2022-05-12 12:55:01}
-*** Builder log completed: {2022-05-12 12:55:24}
-[Warning: Function plot has the same name as a MATLAB builtin. We suggest you rename the function to avoid a potential name conflict.] 
-[> In path (line 109)
-In addpath (line 86)] 
+*** Builder log created: {2022-08-24 12:24:43}
+*** Builder log completed: {2022-08-24 12:25:07}
+[Warning: The following error was caught while executing 'onCleanup' class
+destructor:
```

Comment:
Note errors here. Might just be warnings due to system differences but wanted to flag.

Reply:
Noted. When I removed the outputs and reran in https://github.com/JMSLab/Template/pull/56/commits/9af7f8d289b1be1c3ea5bbdb698cf368deb73936, the errors went away. Not sure where they originated, have not been able to reproduce. My guess is it was some sort of system difference as you noted.